### DISCUSSION TITLE
Duplicated Devices with a Plus Sign

### DISCUSSION THREAD

**Discussion by shawnhaywood** [andrew-codechimp/HA-Battery-Notes#744] [General]
I added the integration and setup automatically detected devices with batteries.  now when I added the recommended battery card, it lists duplicated devices with a plus sign.  The items with a plus sign are devices that were automatically found.  How do I remove the duplicated devices? 
![Screenshot 2024-01-31 192248](https://github.com/andrew-codechimp/HA-Battery-Notes/assets/93692595/4a4ad889-2410-4619-a9cf-ea356d59d43f)

**Comment by andrew-codechimp** [comment]
The new Battery+ sensors are sensors with additional attributes to help making cards with battery types far easier.
You can either use something like Battery State Card as documented [here](https://andrew-codechimp.github.io/HA-Battery-Notes/community/#battery-state-card) which only shows the + batteries or you can just hide the old batteries like [here](https://andrew-codechimp.github.io/HA-Battery-Notes/configuration/#configuration)
